### PR TITLE
Implement `ICTUSNet` format parsing for csv Import/Export

### LINKED ISSUE #18: Replace separator with | instead of ,
The specifications of the ICTUSNet variable registry indicate that the separator should be a pipe.

Moreover, we could add a test to check the number of pipes, as specified in the Technical Interoperability Document D1.3.1 found [here](http://ictusnet-sudoe.eu/pub_type/deliverables/): 

> The character "|" must appear 47 times: the excel file has 48 fields that have to be (all) filled up as follows: all the
variables are mandatory except those in orange. Those variables in green are conditional to the values of the variable above (there is an explanation in the last colum of the excel file "20191002_Variables_ICTUSnet_appweb.xlsx"). If there are unknown values for mandatory variables, you must put 99 only in places permitted. If there are unknown values for non-mandatory or conditional variables, you must leave a blank space between pipes (||).

### PR DESCRIPTION
**Pull request by BPierrick**
## Fixes

- Fixes #18 

## Description

- Parses `ICTUSNet`CSV format to import/export their CSV versions

## Breaking Note

- Previous CSV format handled by the application is now deprecated. There is no way of importing/exporting it anymore.

### CODE REVIEW

**Review by simonvadee — COMMENTED** [review]
I would have used a constant to do that, what do you think ? You would have only one variable to change to do this.

**Review by elsiehoffet-94 — CHANGES_REQUESTED** [review]
Sorry I didn't see this any sooner! The separator we want to change is actually the comma between each variable, not the #. I wrote this issue to make it more clear #18. 
Thanks!

File: app/src/utils/formUtils.ts [hunk 1]
```diff
@@ -15,12 +16,53 @@ import {
...
+import { PatientData, createPatientData } from "../state/patientFormSlice";
 
 const itemSeparator = "#";
 
 type Codes<T extends string = string> = { [code: string]: T };
 
+export const parseIctusFormat = (
+  values: string[]
+): Record<keyof PatientData, string> => {
+  if (values.length !== ICTUS_FORMAT_CSV_COLUMN_KEYS_MAP.length - 2) {
```
Comment (simonvadee) [inline app/src/utils/formUtils.ts:29]: why length - 2?

**Review by simonvadee — APPROVED** [review]
lgtm

**Pull request merged**